### PR TITLE
Add info for Redis Binding create operation

### PR DESCRIPTION
**Pull request by sarsharma**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [x] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Add info for Redis binding create operation
Fix missing value in Kubernetes Binding metadata table

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/redis.md [hunk 1]
```diff
@@ -68,6 +68,27 @@ This component supports **output binding** with the following operations:
 
 - `create`
 
+We can store a record in Redis using the `create` command. This will set a key to hold a value. If a key already holds a value, it will be overwritten.
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/redis.md]:
```suggestion
You can store a record in Redis using the `create` operation. This sets a key to hold a value. If the key already exists, the value will be overwritten.
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/redis.md [hunk 2]
```diff
@@ -68,6 +68,27 @@ This component supports **output binding** with the following operations:
...
+  "data": {
+    "Hello": "World",
+    "Lorem": "Ipsum"
+  }
+}
+```
+
+### Response
+
+An HTTP 204 (No Content) and empty body will be returned if successful.
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/redis.md]:
```suggestion
An HTTP 204 (No Content) and empty body is returned if successful.
```

File: daprdocs/content/en/reference/components-reference/supported-bindings/redis.md (hunk 1 above)
Comment (sarsharma) [reply daprdocs/content/en/reference/components-reference/supported-bindings/redis.md]: Thanks! Updated.

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**